cmd/machine-driver-libvirt/main.go

```diff
@@ -1,10 +1,19 @@
 package main
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/code-ready/machine-driver-libvirt"
 	"github.com/code-ready/machine/libmachine/drivers/plugin"
 )
 
 func main() {
+	if len(os.Args) > 1 {
+		if os.Args[1] == "version" {
+			fmt.Printf("This is a hypervisor plugin binary for CodeReady Containers.\nPlease use this plugin through the main 'crc' binary.\nDriver version: %s\n", libvirt.DriverVersion)
```

Comment:
This 'error' is generic and should actually be given when NO command is given.  The version gets appended when version is requested.

Reply:
when no command is given we error out with this generic (`This is a hypervisor plugin binary for CodeReady Containers.\nPlease use this plugin through the main 'crc' binary`) message, but when version is given we add the version or ONLY display the version. It would make more sense without the previous message